R/MoBiProject.R

```diff
@@ -119,42 +193,112 @@ MoBiProject <- R6::R6Class(
     #' (IC) building block (BB) of each module will be selected. If a module has multiple
     #' IC BBs, it is possible to specify which IC BB to apply by providing a named list,
     #' where the name should be the name of the module and the value the name of the IC BB.
-    #' By setting the value to `NULL`, no IC BB from the specified module will be applied.
+    #' If `NULL`, all modules will use the first IC BB, if available. When providing a list,
+    #' the value can also be set to `NULL`, which means that no IC BB from the specified module
+    #' will be selected.
     #' @param selectedParameterValues By default, the first Parameter Values
     #' (PV) building block (BB) of each module will be selected. If a module has multiple
     #' PV BBs, it is possible to specify which PV BB to apply by providing a named list,
     #' where the name should be the name of the module and the value the name of the PV BB.
-    #' By setting the value to `NULL`, no PV BB from the specified module will be applied.
+    #' If `NULL`, all modules will use the first PV BB, if available. When providing a list,
+    #' the value can also be set to `NULL`, which means that no PV BB from the specified module
+    #' will be selected.
     #'
     #' @returns A `SimulationConfiguration` object.
     createSimulationConfiguration = function(modulesNames, individualName = NULL, expressionProfilesNames = NULL, selectedInitialConditions = NULL, selectedParameterValues = NULL) {
```

Comment:
would rename the 2 last arguments to just `initialConditions` and `parameterValues`